client/src/components/jobPosting/JobPostingForm.js

```diff
@@ -33,17 +33,39 @@ export class JobPostingForm extends Component {
       this.setState({ id: jobPostingId })
       this.props.fetchJobPostingWithStages(jobPostingId)
     }
+
+    if (this.props.location.state && this.props.location.state.mode === 'copy') {
+
+      if (this.props.jobPosting.title) {
+        const jobPosting = this.props.jobPosting
+        this.setState({
+          title: jobPosting.title,
+          content: jobPosting.content
+        })
+      } else {
+        this.setState({
+          mode: 'copy'
```

Comment:
I don't quite understand why the mode is set to 'copy' here, as there doesn't seem to be anything to copy. Wouldn't this just be 'create'?